COPY/etc/httpd/conf.d/manageiq-http.conf

```diff
@@ -15,8 +15,8 @@ RewriteRule ^.*$ https://%{SERVER_NAME}%{REQUEST_URI} [L,R]
 # Enable this section if using HTTP only
 #<VirtualHost *:80>
 #  DocumentRoot /var/www/miq/vmdb/public
+#  Include conf.d/manageiq-redirects-api
 #  Include conf.d/manageiq-redirects-ui
-#  Include conf.d/manageiq-redirects-ws
```

Comment:
Note that I changed the ordering to match the ordering elsewhere...since this is a comment it didn't actually "matter" here